content/en/serverless/google_cloud_run/_index.md

```diff
@@ -1,6 +1,10 @@
 ---
 title: Google Cloud Run
 kind: documentation
+further_reading:
+    - link: 'https://docs.datadoghq.com/serverless/google_cloud_run/'
```

Comment:
I don't think this is the right link?

Reply:
Also, need the heading and shortcode at the bottom of the page for this to show up

Reply:
😬 oops, friday brain